src/atomate2/cp2k/schemas/calc_types/enums.py

```diff
@@ -1,14 +1,13 @@
 """
-Autogenerated Enums for CP2K RunType, TaskType, and CalcType.
-
-Do not edit this by hand. Edit generate.py or run_types.yaml instead.
+Autogenerated Enums for CP2K RunType, TaskType, and CalcType
+Do not edit this by hand. Edit generate.py or run_types.yaml instead
 """
 
 from emmet.core.utils import ValueEnum
 
 
 class RunType(ValueEnum):
-    """CP2K calculation run types."""
+    """CP2K calculation run types"""
```

Comment:
I think these full stops should actually be there?

Reply:
I didn't manually edit those, so they were changed automatically when I built the docs using sphinx. Will fix those!

Reply:
Ah, I remember this. Yes, sphinx causes them to be regenerated. I think this problem could be fixed in the future by adding the correct punctuation in the script that auto generates this file.

Reply:
I've now fixed the script which autogenerates that file, so building the docs and running pre-commit should leave that file unaffected,